packages/terra-date-picker/translations/es-US.json

```diff
@@ -5,9 +5,11 @@
   "Terra.datePicker.closeCalendar": "Cerrar",
   "Terra.datePicker.today": "Hoy",
   "Terra.datePicker.monthLabel": "Mes",
+  "Terra.datePicker.dayLabel": "Día",
   "Terra.datePicker.yearLabel": "Año",
   "Terra.datePicker.date": "Fecha",
   "Terra.datePicker.nextMonth": "Mes siguiente",
   "Terra.datePicker.previousMonth": "Mes anterior",
-  "Terra.datePicker.calendarInstructions": "Para cambiar la selección, use las teclas de flecha. Presione Entrar para seleccionar una fecha. Presione Esc para cerrar el selector de fecha."
+  "Terra.datePicker.calendarInstructions": "Para cambiar la selección, use las teclas de flecha. Presione Entrar para seleccionar una fecha. Presione Esc para cerrar el selector de fecha.",
+  "Terra.datePicker.dateFormatOrder": "2"
```

Comment:
This is really confusing to me. Why aren't we utilizing the date format already included in these translations files?

Reply:
Essentially this allows me to parse the date format without having to worry about localization string variances. Ex: it doesn't matter that the format is `DD/MM/AAAA`, just that it is in day/month/year format. If anyone has any other suggestions I'm not attached to it, so I'm happy to accomplish this another way.